### PR TITLE
Removed AQP 156 Blend Energy GRPD & AQP 157 Blend Energy WLFM + Added…

### PR DESCRIPTION
**Pull request by JappaWakka**
… AQP 157 Sneasel

### CODE REVIEW

File: src/main/resources/cards/202-pokemod_aquapolis.yaml [hunk 1]
```diff
@@ -3782,28 +3782,37 @@ cards:
...
-  pioId: pkmod14-156
-  enumId: BLEND_ENERGY_GRPD_156
-  name: Blend Energy GRPD
-  number: '156'
-  superType: ENERGY
-  subTypes: [SPECIAL_ENERGY]
-  rarity: Rare Holo
-  text: ['This card provides 1 [C] Energy.', 'When this card is attached to a Pokémon, this
-  card provides [G], [R], [P], or [D] but only provides 1 Energy at a time.']
-  artist: Shin-ichi Yoshikawa
```
Comment (axpendix) [inline src/main/resources/cards/202-pokemod_aquapolis.yaml:3795]: So does this mean Blend Energy is discontinued?

Comment (axpendix) [reply src/main/resources/cards/202-pokemod_aquapolis.yaml:3795]: If yes, the numbering seems to have a hole in between, 156 is missing. Do you think we need to shift the subsequent ones? It's not required but just raising a flag.

Comment (JappaWakka) [reply src/main/resources/cards/202-pokemod_aquapolis.yaml:3795]: I don't know, Goku told me to remove those two, but I believe there are still other blend energy cards in that set though

Comment (JappaWakka) [reply src/main/resources/cards/202-pokemod_aquapolis.yaml:3795]: 156 is supposed to be empty to leave a slot for a possible future card

**Pull request merged**